templates/apollo-starter-ts/README.md

```diff
@@ -7,3 +7,25 @@ Starter template using graphback and apollo-server
 docker-compose up -d
 npm start
 ```
+
+### Project structure
+
```

Comment:
Without explanation this makes no sense. I will put typical folder structure. we need the same in docs. We are missing cli in docs at the moment

Reply:
Okay, so close this?

Reply:
Absolutely not. I would never say no for more documentation especialy that we need to document this, however we need to put here more than just result from `three`command. 

I think we should agree on our strategy to document entire project.

We can be:

A) Command line centric
B) Concept centric with workflow. 

I think that we should have some kind of getting started documentation 
where we will go with Init. Pick one of the templates and execute points that were printed in init command. Project structure is cool but I would love to have just more text explaining the steps and how to threat this project.